std_msgs/Header header
string frame_id
float32 cell_size_x
float32 cell_size_y
float32 cell_size_z
ground_segmentation/GridCell[] cells
